charts/stable/projectsend/values.yaml

```diff
@@ -20,12 +20,47 @@ service:
 workload:
   main:
     podSpec:
-      initContainers:
-        initconfig:
-          type: install
-          enabled: true
-          imageSelector: image
+      containers:
+        main:
+          lifecycle:
+            postStart:
+              type: exec
+              command:
+                - /bin/sh
```

Comment:
What difference does it make? (init -> lifecycle)

Reply:
the error says it doesnt write to the file and the file from the last PR is blanked null file. https://github.com/truecharts/charts/actions/runs/6323225204/job/17170520924#step:10:561

Reply:
If the file is only needed at startup as an onetime copy, just mount it as a config map on /app/blablabla and call it a day